third-party/realdds/src/dds-stream-impl.h

```diff
@@ -0,0 +1,42 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <vector>
+
+namespace realdds {
+
+
+class dds_video_stream::impl
```

Comment:
Can you please add a comment in the code why do we need an `impl` class..
We now have several and it's important that the class objective will be documented

Reply:
For all `impl` classes the objective is to hide implementation details from the user (with added benefit of fewer includes in public h files).
See [pimpl](https://en.cppreference.com/w/cpp/language/pimpl) for more details